lib/hexpm_web/router.ex

```diff
@@ -160,6 +160,7 @@ defmodule HexpmWeb.Router do
     for prefix <- ["/", "/repos/:repository"] do
       scope prefix do
         post "/publish", ReleaseController, :publish
+        post "/replace", ReleaseController, :replace
```

Comment:
seeing the scope of work, i think that would be fine too. One thing I was having trouble with is including it in the body of teh request since it seems like the body is expecting a compressed binary (i think)

Reply:
I don't think adding it to the body would work because of the reason you stated, but I think we could it to the URL as `/publish?replace=true`. Plug will be able to handle params from both the body and URL.